build.sbt

```diff
@@ -21,5 +30,24 @@ lazy val core = project
   .settings(name := "quasar-destination-s3")
   .settings(
     performMavenCentralSync := false,
+    quasarPluginName := "s3-dest",
+    quasarPluginQuasarVersion := QuasarVersion,
+    quasarPluginDestinationFqcn := Some("quasar.destination.s3.S3DestinationModule$"),
+    quasarPluginDependencies ++= Seq(
+      "io.argonaut"  %% "argonaut" % ArgonautVersion,
+      "co.fs2" %% "fs2-core" % Fs2Version,
+      "io.monix" %% "monix-catnap" % MonixVersion,
```

Comment:
There's a bit of dependency problem here. `monix-catnap` is only available for cats-effect > 2.0, but quasar is still on cats-effect 1.4.0. It's causing unsafe evictions. I'm thinking about dropping `monix-catnap` and instead one-off it here. So far I haven't seen any problems related to the cats-effect eviction.

Reply:
Cats Effect 2 is binary compatible with Cats Effect 1. Cats as well. I'm considering just bumping us across the board.